### PR TITLE
Adding a FOLLOW feature. Adding silver to report.

### PR DESCRIPTION
**Pull request by MJ659600**

### CODE REVIEW

**Review by Stx69 — APPROVED** [review]

**Comment by phated** [comment]
@jacobrosenthal anything you want done here before merge?

**Comment by jacobrosenthal** [comment]
Nope. Sorry for delay @MJ659600 

In a future pr because I dont want to hold this one up there are many instances of voages -> voyages

**Pull request merged**
